File: dotNetTips.Utility/IO/ProgressState.vb
' ***********************************************************************
' Assembly         : dotNetTips.Utility
' Author           : David McCarter
' Created          : 07-09-2017
'
' Last Modified By : David McCarter
' Last Modified On : 07-09-2017
' ***********************************************************************
' <copyright file="Processor.vb" company="McCarter Consulting - David McCarter">
'     David McCarter - dotNetTips.com © 2017
' </copyright>
' <summary></summary>
' ***********************************************************************
Imports System

''' <summary>
''' Enum ProgressState
''' </summary>
Public Enum ProgressState
    ''' <summary>
    ''' The error
    ''' </summary>
    [Error]
    ''' <summary>
    ''' The deleted
    ''' </summary>
    Deleted
    ''' <summary>
    ''' The moved
    ''' </summary>
    Moved
    ''' <summary>
    ''' The copied
    ''' </summary>
    Copied
End Enum

File: dotNetTips.Utility.Standard/IO/ProgressState.cs
// ***********************************************************************
// Assembly         : dotNetTips.Utility.Standard
// Author           : David McCarter
// Created          : 08-07-2017
//
// Last Modified By : David McCarter
// Last Modified On : 09-16-2017
// ***********************************************************************
// <copyright file="ProgressState.cs" company="dotNetTips.com - David McCarter">
//     dotNetTips.com - David McCarter
// </copyright>
// <summary></summary>
// ***********************************************************************
namespace dotNetTips.Utility.Standard.IO
{
    /// <summary>
    /// Enum ProgressState
    /// </summary>
    public enum ProgressState
    {
        /// <summary>
        /// The error
        /// </summary>
        Error = 0,

        /// <summary>
        /// The deleted
        /// </summary>
        Deleted = 1,

        /// <summary>
        /// The moved
        /// </summary>
        Moved = 2,

        /// <summary>
        /// The copied
        /// </summary>
        Copied = 3
    }
}
